pkg/sdkserver/sdkserver.go

```diff
@@ -547,35 +555,108 @@ func (s *SDKServer) stopReserveTimer() {
 // [Stage:Alpha]
 // [FeatureFlag:PlayerTracking]
 func (s *SDKServer) PlayerConnect(ctx context.Context, id *alpha.PlayerID) (*alpha.Bool, error) {
-	panic("implement me")
+	if !runtime.FeatureEnabled(runtime.FeaturePlayerTracking) {
+		return nil, errors.New(string(runtime.FeaturePlayerTracking) + " not enabled")
```

Comment:
Could be something like:
```suggestion
		return nil, errors.New(fmt.Sprintf("feature gate '%s' is not enabled", runtime.FeaturePlayerTracking))
```

Reply:
We've currently got this code all through localsdk.go as well. Should I change both?

Reply:
In fact, this should `errors.Errorf(...)` - which is much better. I'll make the change here, and come back around to localsdk.go in a later PR.